extensions/measurement-tracking/src/viewports/TrackedCornerstoneViewport.js

```diff
@@ -257,7 +257,7 @@ function TrackedCornerstoneViewport(props) {
           isLocked: false,
           isRehydratable: false,
           studyDate: formatDate(SeriesDate), // TODO: This is series date. Is that ok?
-          currentSeries: SeriesNumber,
+          currentSeries: SeriesNumber || 0,  // TODO - switch entire currentSeries to be UID based or actual position based
```

Comment:
Can't we do this at the displaySet creation level?

Reply:
Done at the display set creation level now.